data/mode.txt

```diff
@@ -1 +1 @@
-sandbox
+Vampire
```

Comment:
what the fug

Reply:
It's not ready for merging yet, I just wanted a middle of the road checkpoint